test/parallel/test-console-instance.js

```diff
@@ -72,7 +72,7 @@ c.log('test');
 c.error('test');
 
 out.write = common.mustCall((d) => {
-  assert.strictEqual('{ foo: 1 }\n', d);
+  assert.strictEqual(d, '{ foo: 1 }\n')
```

Comment:
Please add the semicolon back.

Reply:
doh! sorry about that. pushed that change just up